app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/KanbanActionColumn.jsx

```diff
@@ -24,8 +33,9 @@ const KanbanActionColumn = ({
         {(provided, snapshot) => {
           return (
             <div
-              className={`h-full overflow-x-hidden ${
-                isHovering && !collapsed
+              className={`h-full overflow-x-hidden d:flex d:items-stretch lg:flex lg:items-center lg:flex-col lg:inline-block lg:w-full flex items-center flex-col inline-block w-full 
```

Comment:
Several duplicates on this line. If its on mobile (no letters in front) theres no need to specify the same thing for lg: ? :)